docs/tutorials/control/scuttle-gamepad.md

```diff
@@ -20,73 +20,83 @@ date: "10 August 2022"
+- Select `input_controller` for the component **Type**.
+- Select `gamepad` for the **Model**.
+- Click **Create component**.
+
+![Blank configuration JSON](/tutorials/scuttle-gamepad/gamepad-config.png)
 
 The controller config adds the gamepad controller to your robot.
-However, it does not wire it up to any functionality.
+However, it does not connect it to any functionality.
 This requires a Service.
 
-## Adding a Service
-
-To link the controller's input to the base functionality, we need to add our first `service`.
-Services are the software packages that provide our robots with cool and powerful functionality.
+## Add the Base Remote Control Service
 
-1. Click **Create Service** under **services** at the top of the Viam app ([app.viam.com](https://app.viam.com)).
-2. Enter "Base Remote Control" for **Component** `type`. "Base Remote Control" is a service we provide for driving a rover with a gamepad.
-3. Enter **scuttle_gamepad** for the **Service** `name`.
+Services are software packages that provide robots with higher level functionality.
+To link the controller's input to the base functionality, you need to configure the [Base Remote Control Service](/services/base-rc/):
 
-![Create service builder](/tutorials/scuttle-gamepad/pi-game-create-service.png)
+- Go to the **Services** subtab of your robot's **Config** tab.
+- In the **Create service** panel, click the **Type** dropdown and select `Base Remote Control`.
+- Enter `gamepad_service` for the **Service** **name**.
+- Click **Create service**.
 
-Copy and paste the following into the empty **Attributes** field:
-</OL>
+Copy and paste the following into the empty **Attributes** field, replacing `THE NAME OF YOUR BASE` with your base's name.
 
 ```json {class="line-numbers linkable-line-numbers"}
 {
- "base": "scuttle",
- "input_controller": "gamepad"
+ "base": "THE NAME OF YOUR BASE",
+ "input_controller": "my-gamepad"
 }
 ```
 
-![Blank configuration json](/tutorials/scuttle-gamepad/pi-game-game-config-blank.png)
+<br>
 
-After adding the **Attributes**, your config screen should appear similar to this:
+![Service configuration](/tutorials/scuttle-gamepad/gamepad-service-config.png)
 
-![Service configuration](/tutorials/scuttle-gamepad/pi-game-service-config.png)
+Click **Save config**, then go to the **Control** tab.
 
-Save the configuration and visit the control UI on the Viam app ([app.viam.com](https://app.viam.com)).
+You should see the panel for the gamepad and its connection indicator:
 
-You should see the panel for the Controller Service and its connection indicator.
-This is how your web UI will look.
-Note the green connection indicator:
+![Gamepad input UI showing a "connected" indicator and a list of inputs from all the buttons, for example X=0.0, RY=0.0 and East=0.](/tutorials/scuttle-gamepad/control-tab-input.png)
 
-![Gamepad input UI](/tutorials/scuttle-gamepad/pi-game-controller-panel.png)
+At this point, you should be able to move the rover using the gamepad.
```

Comment:
Instead of saying "you should be able to", instruct them waht to do to test it.

Reply:
```suggestion
Try moving the left joystick or pressing the D-pad to move the rover using the gamepad.
```